go.mod

```diff
@@ -226,7 +226,7 @@ replace (
 	// Use cosmos keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
 	github.com/cometbft/cometbft-db => github.com/crypto-org-chain/cometbft-db v0.0.0-20230921030527-0d47d7537e32
-	github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.46.0-beta2.0.20231009083556-8fbcfbe50690
+	github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.46.0-beta2.0.20231013072415-eec017435467
```

Comment:
Diff
https://github.com/crypto-org-chain/cosmos-sdk/commit/eec017435467

Reply:
Instead of adding the fix in cosmos-sdk (unless it got merged upstream), would it be possible to add the fix in the upgrade handler instead?

Reply:
seems not a blocker right?
https://github.com/cosmos/cosmos-sdk/pull/18107/files
https://github.com/cosmos/cosmos-sdk/pull/18113/files

Reply:
okay nvm it seems the PR got approved